### PR TITLE
Nostree basic implementation

### PR DESCRIPTION
**Pull request by gzuuus**
Hi, I really liked the client! I've added basic support for link lists, these were already in the profile lists but empty, I've also added nostree to show profiles outside of nostrudel.

### CODE REVIEW

**Comment by hzrd149** [comment]
This looks awesome, and thanks for adding the helper method and sticking with my coding style.
Do you have a few example npubs that I can test with? just want to make sure its working before merge

**Comment by gzuuus** [comment]
Nice! glad it looks good for you :)
Yes, you can try mine: npub1gzuushllat7pet0ccv9yuhygvc8ldeyhrgxuwg744dn5khnpk3gs3ea5ds
Or you can see more example pubkeys in the explore section of nostree ( nostree.me/explore ).

File: src/views/lists/components/list-card.tsx [hunk 1]
```diff
@@ -82,6 +89,11 @@ function ListCardRender({ event, ...props }: Omit<CardProps, "children"> & { eve
             </Flex>
           </>
         )}
+        {references.length > 0 && (
+          <>
+            <Text>References ({references.length}):</Text>
```
Comment (hzrd149) [inline src/views/lists/components/list-card.tsx:94]:
```suggestion
            <Text>References ({references.length})</Text>
```

File: src/views/lists/list-details.tsx [hunk 2]
```diff
@@ -7,7 +7,7 @@ import { ArrowLeftSIcon } from "../../components/icons";
 import { useCurrentAccount } from "../../hooks/use-current-account";
 import { useDeleteEventContext } from "../../providers/delete-event-provider";
 import { parseCoordinate } from "../../helpers/nostr/events";
-import { getEventsFromList, getListName, getPubkeysFromList } from "../../helpers/nostr/lists";
+import { getEventsFromList, getListName, getPubkeysFromList, getReferencesFromList } from "../../helpers/nostr/lists";
 import useReplaceableEvent from "../../hooks/use-replaceable-event";
 import { EventRelays } from "../../components/note/note-relays";
 import UserCard from "./components/user-card";
```
Comment (hzrd149) [inline src/views/lists/list-details.tsx:13]:
```suggestion
import UserCard from "./components/user-card";
import OpenGraphCard from "../../components/open-graph-card";
```

File: src/views/lists/list-details.tsx [hunk 3]
```diff
@@ -100,6 +101,22 @@ export default function ListDetailsView() {
...
+      {references.length > 0 && (
+        <>
+          <Heading size="md">References</Heading>
+          <Divider />
+          <TrustProvider trust>
+            <Flex gap="2" direction="column">
+              {references.map(({ url, petname }) => (
+                <a href={url} target="_blank" rel="noopener noreferrer">
+                  <Button>{petname}</Button>
+                </a>
```
Comment (hzrd149) [inline src/views/lists/list-details.tsx:114]:
```suggestion
                <>
                  {petname && <Heading size="md">{petname}</Heading>}
                  <OpenGraphCard url={new URL(url)} />
                </>
```

**Review by hzrd149 — COMMENTED** [review]
Added a few suggestions to use the `OpenGraphCard` component instead of a `Button` for the references

Before:
Screenshot from 2023-09-16 14-57-52

After:
Screenshot from 2023-09-16 14-57-01

**Comment by gzuuus** [comment]
Looks amazing!

**Comment by gzuuus** [comment]
Think its ready to merge? what do you think?

**Comment by hzrd149** [comment]
Applied suggestions and merged :+1:

**Pull request merged**